WISE-Community/WISE
# Issue 1218 add student names to export if owner

Test to make sure the run owner has the option to include student names in the export and that it exports the student names properly.
1. Log in as a teacher that owns a run.
2. Open the Classroom Monitor for the run.
3. Go to the Data Export view.
4. Make sure these export views below have the checkbox option to include student names and that it properly includes the student names in the actual exports.

Export One Workgroup Per Row
Export Latest Work
Export All Work
Export Events
Export Component Data (this only works with Match components)
Export Raw Data

Test to make sure shared run owners do not have the option to include student names in the export.
1. Log in as a teacher that is the shared owner of a run.
2. Open the Classroom Monitor for the run.
3. Go to the Data Export view.
4. Make sure these export views below do not have the checkbox option to include student names and that it does not include the student names in the actual exports.

Export One Workgroup Per Row
Export Latest Work
Export All Work
Export Events
Export Component Data (this only works with Match components)
Export Raw Data
